README.md

```diff
@@ -89,6 +89,7 @@ docker run \
 | GITHUB_OAUTH2_CLIENT_ID                      | GitHub oauth2 client ID                                                                                                         |    no    |                        |                                 |
 | GITHUB_OAUTH2_CLIENT_SECRET                  | Gihub oauth2 client secret                                                                                                      |    no    |                        |                                 |
 | BASIC_AUTH_USER_DETAILS_FILE_PATH            | File location for basic auth user details                                                                                       |    no    |                        |         /src/.htpasswd          |
+| BASIC_AUTH_USER_DETAILS                      | Contents of basic auth file. Use as an alternative for BASIC_AUTH_USER_DETAILS_FILE_PATH                                        |    no    |                        |                                 |
```

Comment:
Update the description to point either BASIC_AUTH_USER_DETAILS_FILE_PATH or BASIC_AUTH_USER_DETAILS can be present at any given point of time